cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -121,7 +121,9 @@ func initConfig() (*viper.Viper, error) {
 		viper.SetConfigFile(viper.GetString(configOpt))
 	} else {
 		viper.AddConfigPath(projutil.MustGetwd())
-		// using SetConfigName allows users to use a .yaml, .json, or .toml file
+		// Note that viper allows other extensions as  .json, or .toml file, however,
+		// these other formats are deprecated in the SDK.
+		// By using SetConfigName allows users to use a .yaml
```

Comment:
```suggestion
		// By using SetConfigName allows users to use  diff extensions. 
		// todo(camilamacedo86): Check if we can replace this configuration and make the things easier for the future versions since from 0.16 we will need just support the YAML format. 
```